app/assets/javascripts/angular/controllers/login.js.erb

```diff
@@ -0,0 +1,16 @@
+app.controller('LogInCtrl', ['$scope', '$auth', '$location', function($scope, $auth, $location){
+  $scope.login = function () {
+    $auth.submitLogin($scope.loginForm)
+    .then(function(resp) {
+      // handle success response
+      console.log(resp);
```

Comment:
just a reminder. Please remove these console.log when you are done with the controller and made sure the function works